### DISCUSSION TITLE
Acteur Journal : Recherche des évènements par rapport à un texte

### DISCUSSION THREAD

**Discussion by JeremieDENAEGEL** [witsa/synapps#694] [Q&A]
Bonjour,

Je cherche à faire un filtre pour pouvoir rechercher un texte dans la colonne Libellé ou Etat du journal comme dans le menu "Exploitation>Journal"

Est-ce-que quelqu'un aurait déjà fait quelque chose de similaire ? je ne trouve pas la possibilité dans l'acteur

![journal](https://github.com/witsa/synapps/assets/50445069/89846fe1-1783-44ca-a3b2-8a5176d4cb66)

**Comment by manufitoussiwit** [comment]
Salut,
Effectivement, ce n'est pas possible pour l'instant avec l'acteur journal des variables. 
C'est possible par contre l'acteur journal des reflets mais limité donc aux seuls évènements liés à des reflets.
A noter aussi que ça cherche uniquement dans le nom des ressources.

**Reply by JeremieDENAEGEL** [reply]
Pourras-tu prévoir cette évolution ?

Dans le développement que je prépare, j'ai des ressources scripts qui génèrent différents évènements relatifs aux sondes associées et le filtre texte du journal des reflets ne réponds pas pleinement aux recherches car juste avant de créer l'évènement journal je change le .Title du script avant de reprendre son libellé original.

**Reply by manufitoussiwit** [reply]
C'est noté! On va étudier ça. 
Je reviendrai ici rapidement pour tenir au courant de la faisabilité et si oui quand ce sera dispo.
